app/models/shipit/task.rb

```diff
@@ -29,8 +29,11 @@ class Task < ActiveRecord::Base
     scope :success, -> { where(status: 'success') }
     scope :completed, -> { where(status: COMPLETED_STATUSES) }
     scope :active, -> { where(status: ACTIVE_STATUSES) }
+    scope :not_active, -> { where(status: COMPLETED_STATUSES + UNSUCCESSFUL_STATUSES) }
     scope :exclusive, -> { where(allow_concurrency: false) }
     scope :unsuccessful, -> { where(status: UNSUCCESSFUL_STATUSES) }
+    scope :current_week, -> { where("created_at > ?", 7.days.ago) }
+    scope :previous_week, -> { where(created_at: 14.days.ago..7.days.ago) }
```

Comment:
These `ago` means "last week" is a sliding window. I guess it's fine, but as always with stats, it might throw people off.